src/com/couchbase/perf/shared/database/PerfDatabase.groovy

```diff
@@ -68,6 +68,24 @@ class PerfDatabase {
         // JSONB indexes
         execute(sql, env, "CREATE INDEX IF NOT EXISTS idx_runs_params ON runs USING gin (params)")
         execute(sql, env, "CREATE INDEX IF NOT EXISTS idx_runs_events_params ON run_events USING gin (params)")
+        // New Jobs table to hold an overlying job for grouping runs
```

Comment:
Please slot this into the existing code a little better.  This is in the middle of creating indexes.  Create a separate "// FIT-as-a-Service" section maybe.